### PR TITLE
run rustfmt on libsyntax_ext/deriving folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
cc @nrc

File: src/libsyntax_ext/deriving/clone.rs [hunk 1]
```diff
@@ -46,8 +48,8 @@ pub fn expand_deriving_clone(cx: &mut ExtCtxt,
             match annitem.node {
                 ItemKind::Struct(_, Generics { ref ty_params, .. }) |
                 ItemKind::Enum(_, Generics { ref ty_params, .. })
-                    if ty_params.is_empty()
-                        && attr::contains_name(&annitem.attrs, "derive_Copy") => {
+                    if ty_params.is_empty() &&
+                    attr::contains_name(&annitem.attrs, "derive_Copy") => {
```
Comment (brson) [inline src/libsyntax_ext/deriving/clone.rs]: This indentation is pretty strange. Why is the continuation line at the same level is `if`?

**Comment by brson** [comment]
@nrc I'm not sure how to review these patches. They all seem to have parts that look like they might want to be tweaked. With the actual style conventions in flux, are we happy to land these with formatting that is sometimes not very attractive? What's the goal with rustfmting the compiler?

**Comment by brson** [comment]
Triage: no change since my previous comment.

File: src/libsyntax_ext/deriving/clone.rs (hunk 1 above)
Comment (nrc) [inline src/libsyntax_ext/deriving/clone.rs]: It's a bug, I filed https://github.com/rust-lang-nursery/rustfmt/issues/1099 for this

**Comment by nrc** [comment]
@brson I would like to not block patches like this on perfect formatting. I would block if we lose formatting that won't easily be recreated (e.g., a matrix alignment) or if the formatting is really gross (i.e., you would be unhappy working with such code), c.f, slightly off (such as the alignment here). Basically, I think rustfmt is going to get better and these things will naturally get better over time, but if we don't get started, we never will.

Given that, do you think this PR just needs a rebase to land, or are there other blocking issues with it? If there, it would be nice (but not essential) to file rustfmt issues and close this PR until rustfmt is fixed.

**Comment by brson** [comment]
@nrc I'm fine landing this one, though I guess @srinivasreddy will need to redo the patch at this point. Sorry for the delay @srinivasreddy.

**Comment by srinivasreddy** [comment]
@brson  Let's land this cc @nrc

**Comment by brson** [comment]
Thanks @srinivasreddy

**Comment by alexcrichton** [comment]
@bors: retry

* #34595

**Comment by alexcrichton** [comment]
@bors: retry

* https://github.com/rust-lang/rust/issues/33434

**Pull request merged**